tests/test-utils.tsx

```diff
@@ -4,9 +4,16 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { RouterContext } from 'next/dist/shared/lib/router-context';
 import { NextRouter } from 'next/router';
-import { ThemeProvider } from '@material-ui/core';
+import { ThemeProvider, Theme, StyledEngineProvider } from '@mui/material';
 import theme from '../lib/theme';
 
+
+declare module '@mui/styles/defaultTheme' {
```

Comment:
I saw this in another file `pages/_app.tsx`. We should only need to declare this once right?

Reply:
that was already there, the codemod just updated it, so I can see what happens if I take it out.

Reply:
Oh I see where it's needed after a review of the full file.  If you choose to use the custom wrapper defined within the file it needs to be there, but we aren't using it on any of our tests so taking it out would break anything unless someone wanted to use that custom wrapper.